src/wasm-interpreter.h

```diff
@@ -1105,7 +1114,20 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
     Literal c = flow.getSingleValue();
     switch (curr->op) {
       case Bitselect:
+      case LaneselectI8x16:
+      case LaneselectI16x8:
+      case LaneselectI32x4:
+      case LaneselectI64x2:
         return c.bitselectV128(a, b);
+
+      case RelaxedFmaVecF32x4:
+        return a.addF32x4(b.mulF32x4(c));
```

Comment:
As you added `fma()` methods to Literal, I was expecting to see them used here?

Reply:
(This was resolved, but I still don't understand the code in its current form?)

Reply:
fma() methods on Literal work on a single lane, in the latest commit i changed this to use a.relaxedFma32x4(b, c), which calls those fma() methods.

Reply:
I see, thanks!